src/component/MapMarkerIcon.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+const MapMarkerIcon = (): JSX.Element => (
+  <svg
+    width="32px"
+    height="52px"
+    viewBox="0 0 32 52"
+    version="1.1"
+    xmlns="http://www.w3.org/2000/svg"
+    xmlnsXlink="http://www.w3.org/1999/xlink">
+    <defs>
+      <filter
+        x="-42.9%"
+        y="-300.0%"
+        width="185.7%"
+        height="700.0%"
+        filterUnits="objectBoundingBox"
+        id="filter-1"
+      />
+    </defs>
+    <g
+      id="location"
+      stroke="none"
+      strokeWidth="1"
+      fill="none"
+      fillRule="evenodd">
+      <g id="pin-copy-2">
+        <ellipse
+          id="shadow"
+          fill="#000000"
+          opacity="0.800000012"
+          filter="url(#filter-1)"
+          cx="16"
+          cy="46"
+          rx="7"
+          ry="1"
+        />
+        <path
+          d="M15.9981139,0 C20.1340105,0 24.2713477,1.53717334 27.4262114,4.61293808 C33.7359387,10.7630495 33.3095279,20.347694 27.4262114,26.8863529 L26.174361,28.2877277 C20.9310906,34.2007901 18.0099826,38.091836 15.9981139,42 C13.8430107,37.791208 10.613237,33.6022689 4.57433813,26.8863529 C-1.30897841,20.347694 -1.73682979,10.7630495 4.57433813,4.61293808 C7.7292018,1.53717334 11.8650984,0 15.9981139,0 Z M16.0657464,10.8717672 C13.2952057,10.8717672 11.0444238,13.0895872 11.0444238,15.8145918 C11.0444238,18.5373718 13.2952057,20.7551919 16.0657464,20.7551919 C18.8340272,20.7551919 21.0848091,18.5373718 21.0848091,15.8145918 C21.0848091,13.0895872 18.8340272,10.8717672 16.0657464,10.8717672 Z"
+          id="selectable"
+          fill="#0000BF"
+        />
+      </g>
+    </g>
+  </svg>
+);
+
+export default MapMarkerIcon;
```

Comment:
If you're ok with static content, a more straight forward way would be to save the svg as is, and then webpack can import it like so: `import { ReactComponent as Icon } from "../../images/icon.svg";`

Reply:
cool, thanks!